new-log-viewer/src/services/LogFileManager/index.ts

```diff
@@ -254,6 +276,84 @@ class LogFileManager {
         };
     }
 
+    /**
+     * Searches for log events based on the given search string.
+     *
+     * @param searchString
+     * @param isRegex
+     * @param isCaseSensitive
+     */
+    startQuery (searchString: string, isRegex: boolean, isCaseSensitive: boolean): void {
+        this.#queryId++;
```

Comment:
Can we move this increment after the call to `this.#searchChunkAndScheduleNext`? That way the initial value `0` can be used.